lib/chai/core/assertions.js

```diff
@@ -2013,6 +2048,11 @@ module.exports = function (chai, _) {
    *     expect([1, 2, 3]).to.have.lengthOf(3);
    *     expect('foo').to.have.lengthOf(3);
    *
+   * When the target is a map or set, asserts that the target's `size` property.
+   *
+   *     expect(new Set([1, 2])).to.hove.lengthOf(2);
+   *     expect(new Map([['a', 1], ['b', 2]])).to.hove.lengthOf(2);
```

Comment:
There's a typo in these examples: "hove" instead of "have".

Reply:
Thanks!